FusionAuth/fusionauth-quickstart-javascript-react-web
# 32 Update Styling for Authenticated Pages

### What is this PR and why do we need it?
This PR makes the styling and content updates to the pages accessed in the authenticated state.  This removes the Make Change page and combines it into the Account page. The content should now be responsive to any screen port width and height.

**To Test:**

Run `npm run dev` from the Quickstart app
Login as a registered user
Ensure overall functionality of the application has not changed

Pre-Merge Checklist (if applicable)
[x] Unit and Feature tests have been added/updated for logic changes, or there is a justifiable reason for not doing so.
